webpack.config.js

```diff
@@ -0,0 +1,132 @@
+/**
```

Comment:
https://czi-edu.slack.com/archives/C0310HJ9WEN/p1657052534744609?thread_ts=1657047516.904539&cid=C0310HJ9WEN

Reply:
I don't see what this Webpack config has to do with theming 🤔 

Looks like it's a generic config to process
- Sass
- Icon.js
- SVGs